packages/strapi/lib/middlewares/logger/index.js

```diff
@@ -25,14 +26,31 @@ module.exports = strapi => {
     initialize() {
       const { level, exposeInContext, requests } = strapi.config.middleware.settings.logger;
 
-      if (level) {
-        strapi.log.level = level;
+      const logLevels = ['fatal', 'error', 'warn', 'info', 'debug', 'trace'];
```

Comment:
Can you use the levels available at strapi.log.levels to avoid having to maintain this list :)

const logLevels = Object.keys(strapi.log.levels.values);

Reply:
Good idea.